app/models/agents/pushover_agent.rb

```diff
@@ -84,6 +86,7 @@ def receive(incoming_events)
             sound
             retry
             expire
+            html
           ].each do |key|
             if value = String.try_convert(interpolated[key].presence)
               case key
```

Comment:
Add the following lines here to accept `true` as a value for `html`:
```ruby
when 'html'
  case value
  when 'true', '1'
    value = '1'
  else
    next
  end
```

Reply:
I did mostly this, but explicitly set `value = '0'` in the `else` instead of just going to the `next` key.